app/src/main/java/com/koalatea/thehollidayinn/softwareengineeringdaily/PlaybackControllerActivity.java

```diff
@@ -171,9 +171,14 @@ protected void onMediaItemSelected(MediaBrowserCompat.MediaItem item, boolean is
             MediaControllerCompat.TransportControls controls = controller.getTransportControls();
 
             if (isPlaying) {
```

Comment:
Is this variable incorrectly assigned then? It seems the isPlaying should determine wether or not to pause. The fix looks correct after this though!

Reply:
The isPlaying variable actually refers to whether the new media being played is the same as the old. If they are the same, we assume they don't want to restart the episode so we pause. I figure if we were pausing, might as well make it a toggle.

The variable is a little poorly named. I just left it at what it was originally. Something like isSameMedia is probably more descriptive.

Reply:
I figured. This is a good time to rename then. Can you update that?